Assets/MRTK/Examples/Demos/Input/Scenes/InputActions/InputActions.MixedRealityControllerMappingProfile.asset

```diff
@@ -2729,7 +2729,7 @@ MonoBehaviour:
     - id: 5
       description: Axis1D.PrimaryHandTrigger Press
       axisType: 3
-      inputType: 10
+      inputType: 60
```

Comment:
question: what is this change?

Reply:
This was a result of aligning the existing Oculus Touch OpenVR controller with the button types used in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8088. This change specifically changes from `TriggerPress` to `GripPress`, and the others change from the generic `ButtonPress` to the more descriptive `Primary`/`SecondaryButtonPress`.

The underlying OpenVR controller support works with either, so even if older profiles aren't updated to the new input types, it will still read the correct data.

TriggerPress vs GripPress: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ca24b323de3e015feb7d3e46e7b0186783129b5e/Assets/MRTK/Core/Providers/UnityInput/GenericJoystickController.cs#L186

Buttons all fallback to: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ca24b323de3e015feb7d3e46e7b0186783129b5e/Assets/MRTK/Core/Providers/UnityInput/GenericJoystickController.cs#L149-L150